backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/MetadatenDs.java

```diff
@@ -91,4 +92,20 @@ public void setAttributeOfSimpleMetadatum(
               }
             });
   }
+
+  /**
+   * It returns the value for a specific metadata from a single element referenced by its eid for a
+   * specific date.
+   *
+   * @param metadatumSingleElement the enum metadatum
+   * @param eid the eid of the single element
+   * @param date the selected date
+   * @return value of the requested metadata element or empty if it doesn't exist
+   */
+  public Optional<String> getMetadatenDsSingleElement(
```

Comment:
Not happy with this naming. What about aligning the name to what we already had? The abstract class `Metadaten` is defining `getSimpleMetadatum` for extracting the value of a simple metadatum (meaning no child element) and maybe we need to rename that method to `getFrameSimpleMetadatum` ?  So that we can name this new method `getSingleElementSimpleMetadatum`?

Reply:
Agree!